### ISSUE TITLE
Mini Ex 5 Feedback

### ISSUE THREAD

**Issue by MartinfraDD** [hele4924/mini-ex#11]
Hi!

I think this game could be classified as a prototype or a proof-of-concept for what would be a really fun game. In the game you control the tongue of a frog by moving your mouse cursor around the screen, in an attempt to catch flies for the hungry frog. I think the game does a really good job at communicating the objective clearly. When I opened the game, it quickly became clear what the objective of the game was, and the process of discovering that is something that is super fun.

Code-wise the whole thing looks pretty clean, I think it's written in a way that's easy to read and understand without any headaches, and it looks like the implementation of classes works well too! 

Visually the game works super well, it's got this simple and cartoony style which makes it super fun to look at, while also being quite easy to understand early on. It's super cool how a fly can be constructed from what is just three circles. I think the idea of this game is so much fun, and while it's a shame that it could not be completed in time, I think what is here is already looks super solid, and I'm sure a lot of fun stuff could be done with the concept. 

In summary I really like the concept this game presents, and I think it's a super fun and creative idea. Since the game is unfinished, I'm sure a lot of my suggestions are things that were already supposed to be implemented into the game. For example having the flies' wings flutter, or perhaps some background music.  I'm not sure how I'd manage to code a full game like that, so I think it's super cool to see how much ambition went behind it. It'd be super fun to see this idea developed further, but I guess we are all busy at this time haha

Overall this was super fun, thanks for making it!
